### PR TITLE
Feature/cli cleanup

### PR DESCRIPTION
**Pull request by jschra**
This PR needs to be reviewed after PR #8

### CODE REVIEW

File: utils/init_functions_aws.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# -- Base Python imports
+import logging
```
Comment (Jarvin-M) [inline utils/init_functions_aws.py:2]: A same file is called `initialize_functions_aws.py` in https://github.com/jschra/grater_expectations/pull/8/files#diff-553951b6536ddeb7a8b6496a0bb27018c2eaec15681a19b8b48a55a0c0fd0b2d. Could you consolidate the two?

File: utils/init_functions_config.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    """Function to check if a configuration file already exists and ensure that the user
+     wants it to be overwritten, if that is the case"""
+    if config_name in os.listdir(project_root):
+        logging.info(
+            f"A Grater Expectations configuration file ({config_name}) already exists "
+            "in this directory. Are you sure you want to initialize a new file and "
+            "overwrite the existing one (y/[n])?"
+        )
+        response = input("Input: ")
+        if response in ["y", "Y", "yes", "Yes", "YES"]:
```
Comment (Jarvin-M) [inline utils/init_functions_config.py]: I would do:
`if response.lower() in ["y", "yes"]:`
to cater for different combinations of yes e.g YeS

**Review by Jarvin-M — CHANGES_REQUESTED** [review]
Have a look at `init_functions_aws.py` vs `initialize_functions_aws.py` in #8

File: utils/init_functions_aws.py (hunk 1 above)
Comment (jschra) [reply utils/init_functions_aws.py:2]: This should be a rename. Not sure why it is not turning up as such in the PR, let's see what happens upon merge

File: utils/init_functions_config.py (hunk 2 above)
Comment (jschra) [reply utils/init_functions_config.py]: Done, great catch!

**Review by Jarvin-M — APPROVED** [review]

**Review by nancy-from-newzealand — APPROVED** [review]

**Pull request merged**